### ISSUE TITLE
Can we add sub view as this image?

### ISSUE THREAD

**Issue by mengheangrat** [kitasuke/PagingMenuController#325]
<img width="390" alt="screen shot 2017-01-18 at 5 25 03 pm copy" src="https://cloud.githubusercontent.com/assets/10340553/22060681/5ae94c8c-dda4-11e6-8ef3-8640ce6bb1b7.png">

**Comment by kitasuke** [comment]
No. Is this a static image? if so, maybe I can add header view option for that case.
